build-wheelhouse/action.yml

```diff
@@ -0,0 +1,62 @@
+name: "Build the wheelhouse of a Python library"
+description: "Generate a Zip aritfacts for the wheelhouse of a Python library"
+
+inputs:
+  library-name:
+    description: 'Name of the Python library'
+    required: true
+    type: string
+  library-namespace:
+    description: 'Namespace of the Python library'
+    required: true
+    type: string
+  operating-system:
+    description: 'Name of the operating system'
+    required: true
+    type: string
+  python-version:
+    description: 'Desired Python version for build'
+    required: true
+    type: string
+
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: "Install Git and clone project"
+      uses: actions/checkout@v3
+  
+    - name: "Set up Python ${{ inputs.python-version }}"
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ inputs.python-version }}
+
+    - name: "Update pip and install dependencies"
+      shell: bash
+      run: pip install --upgrade pip build wheel
+
+    - name: "Install the library"
+      shell: bash
+      run: pip install .
+
+    - name: "Verify libray is properly installed and get its version number"
+      run: |
+        echo "library_version=$(python -c "from ${{ inputs.library-namespace }} import __version__; print(__version__)")" >> $GITHUB_ENV
+
+    - name: "Generate wheelhouse"
+      shell: bash
+      run: python -m pip wheel . -w wheelhouse
+
+    - name: "Zip wheelhouse"
```

Comment:
I thought we were going to proceed with ``cibuildwheel``

https://github.com/pyansys/pymapdl-reader/blob/8b0ace3f05e71b26fe632ab53538f2053bbee453/.github/workflows/testing-and-deployment.yml#L166-L167

Reply:
Hmm... I may be confused. I thought the wheelhouse packaged all the dependencies so you do not need to install those from an index while the cibuildwheel generated wheels for desired platform. Does the cibuildwheel also include all the dependencies?

Reply:
Ah yeah... that's true. But I thought maybe cibuildwheel was also able to pack all dependencies. Let me check...

Reply:
Yeah, looks like it only builds the specific wheel, not the entire wheelhouse. Resolving comment.